action/sealedenvelope_test.go

```diff
@@ -198,8 +199,42 @@ func createSealedEnvelope(chainID uint32) (SealedEnvelope, error) {
 
 	cPubKey, err := crypto.HexStringToPublicKey(_publicKey)
 	se := SealedEnvelope{}
+	if err != nil {
+		return se, err
+	}
+
 	se.Envelope = evlp
 	se.srcPubkey = cPubKey
-	se.signature = _signByte
+	se.signature = _validSig
+	if err = se.LoadProto(se.Proto()); err != nil {
+		return se, err
+	}
 	return se, err
 }
+
+func createSealedEnvelopeWithSignature(chainID uint32, sig []byte) (SealedEnvelope, error) {
```

Comment:
we don't need to add this func? if there's a need (for test) about signature, just directly set the signature (as done in tests now)